backend/schema.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE tokens(
+  id serial PRIMARY KEY,
+  token_value varchar(255)
```

Comment:
I think we should make sure that the `token_value` is `UNIQUE`

`token_value  VARCHAR(255) UNIQUE NOT NULL,`